RELEASE.md

```diff
@@ -1,24 +1,63 @@
+# Release 1.5.0
+
+The 1.5 minor series tracks TensorFlow 1.5.
+
+## Highlights
+
+- New Custom Scalars dashboard, which can display configurable custom line and
+  margin charts based on the same data as the regular Scalars dashboard. See
+  for details: https://github.com/tensorflow/tensorboard/tree/1.5/tensorboard/plugins/custom_scalar
+- Many projector plugin enhancements thanks to @francoisluus, which enable
```

Comment:
Optimal sentences like these should generally do the following, generally in this exact order:

1. Focuses on communicating benefit to user
2. Credits contributor in way that creates hyperlink (pagerank!)
3. Credits employer generous enough to approve creating that contribution (if applicable)

For example:

```markdown
- Projector plugin enhancements, enabling cognitive-assisted labeling via
  semi-supervised t-SNE (by
  [@francoisluus](https://github.com/francoisluus) from
  [@ibm-research](https://github.com/ibm-research))
```

Note: I'm not sure why GitHub's markdown View button didn't make those `@` tags links automatically?

Reply:
Also I'm still not 100% sure on the best choice of words re: 'by' and 'from'. My main focus here is making sure we give proper credit, but keep it succinct, because the primary emphasis is always user benefit.

Reply:
Also another thing we can do with these notes, is punt all the `#123` link to the end of the sentence after a colon. That might allow people to scan over the release notes quicker.

Reply:
So... this is why I had asked you to take a look at the release notes in the PR, before I submitted it and made the official 1.5.0 tag.  I can make some edits this once, but in general I don't want to be in the position of wordsmithing the release notes after the official release goes out and creating divergence in the release notes content across branches, unless there's an actual error or crucial omission.

Re: the contributor linking, GitHub doesn't interpret GitHub-specific markdown links (issue numbers, @ handles) within ordinary markdown file views since it doesn't know whether that file actually has any connection to GitHub per se (other than being hosted on it).  The @ handles do get interpreted within the actual Code > Releases page on GitHub.  Personally, I think that amount of linking is sufficient and that having to manually construct the handle links just to get it to show up in RELEASE.md is both extra toil and makes the file significantly less readable as raw markdown.  For comparison, tensorFlow's relnotes just use contributor names and don't link to them at all.

Re: by vs from, I think that's down in the weeds.  The important thing is giving credit, not the precise word we use to do it.

Re: links at the ends of sentences, I thought it was useful to show the exact issue for sub-features, but if you prefer I can keep issue links all at the end of the line.  I'd rather stick with parens than colons though since parens are what we already have been using and they're what GitHub automatically does with submitted PRs so it requires less manual editing of the commit messages when I copy them over.